src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -1,18 +1,25 @@
 describe('Facility Search', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/v0/maintenance_windows', []);
+  });
+
   it('displays search results header after searching', () => {
     cy.visit(
       '/find-locations/?address=new%20york&context=New%20York%2C%20New%20York%2C%20United%20States&facilityType=health&location=40.7648%2C-73.9808',
     );
 
-    cy.get('#facility-search-results').should('exist');
     cy.injectAxe();
     cy.axeCheck();
+
+    cy.get('#facility-search-results').should('exist');
   });
 
   it('does not show search result header if no results are found', () => {
     cy.visit('/find-locations?fail=true');
-    cy.get('#facility-search-results').should('not.exist');
     cy.injectAxe();
     cy.axeCheck();
+
+    cy.get('#facility-search-results').should('not.exist');
```

Comment:
Even before without the fix, this test would pass before the page even fully loaded, so it might be more accurate to assert the existence of other elements before this.

Reply:
Does moving the line below the axe check ensure that the page is fully loaded?
Is there a better way of ensuring this?

Reply:
It doesn't ensure the page is fully loaded, but I think it's just best to do the `injectAxe` right after visiting a page. And you can put `axeCheck` wherever you'd like; I just kept it next to the `injectAxe` for now.

I suppose it would work best before and after certain actions that affect the page.

Reply:
If you want a good way to ensure the page is loaded, I suppose you can [wait for certain requests to get responses](https://docs.cypress.io/api/commands/route.html#Without-Stubbing) or assert that certain elements exist (maybe the map or search bar?).